### PR TITLE
Community meetings

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: mkdocs/docs/contributing/overview.md [hunk 1]
```diff
@@ -35,16 +35,25 @@ We welcome many different types of contributions including:
...
+Everyone is welcome to join our community meetings. You never need an
+invite to attend. In fact, we want you to join us, even if you don’t have
 anything you feel like you want to contribute. Just being there is enough!
 
-You can find out more about our meetings [here](../../getting-started/help). We welcome everyone to join! Over time, we hope that you feel comfortable voicing your opinions, giving
+Our community meetings are on Wednesday at 1PM EST/EDT. You can find the video
+conference link [here](https://meet.google.com/hys-ubrv-bsd). You can find the
+agenda [here](https://drive.google.com/drive/folders/1khHMD7JKt-kNAkbLgiIfwDbJYAhHMkyg?usp=sharing) 
+as well as the meeting notes [here](https://drive.google.com/drive/folders/1rVrheoO-nfUI7JdPEgCqM39rl59KrPBk?usp=sharing).
+If you would like to attend a meeting, please list yourself as an attendee in
```
Comment (sriumcp) [inline mkdocs/docs/contributing/overview.md]: Please consider listing yourself as an attendee in the agenda. If you have a topic you would like to discuss, please put it that down along with the amount of time needed for this discussion.
***
We don't want to create another step before people can attend meetings. Having to list my name in a list might make me think twice about attending the meeting in the first place.

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor change requested

File: mkdocs/docs/contributing/overview.md (hunk 1 above)
Comment (Alan-Cha) [reply mkdocs/docs/contributing/overview.md]: Do we not want a list of attendees then? What about the adding topics to the agenda? Is that something we still want?

Comment (Alan-Cha) [reply mkdocs/docs/contributing/overview.md]: Do we want to disregard the list of attendees for now and add it in later when we have more regular members?

Comment (sriumcp) [reply mkdocs/docs/contributing/overview.md]: @Alan-Cha Disregarding for now, and adding it later when things need to be managed better sounds like a good idea. For now, it is ok to have people showing up (we can streamline the agenda at the start of the meetings).

Comment (Alan-Cha) [reply mkdocs/docs/contributing/overview.md]: Okay sounds good to me

Comment (Alan-Cha) [reply mkdocs/docs/contributing/overview.md]: Fixed! Let me know if this change is satisfactory

File: mkdocs/docs/contributing/overview.md [hunk 2]
```diff
@@ -35,16 +35,24 @@ We welcome many different types of contributions including:
...
+invite to attend. In fact, we want you to join us, even if you don’t have
 anything you feel like you want to contribute. Just being there is enough!
 
-You can find out more about our meetings [here](../../getting-started/help). We welcome everyone to join! Over time, we hope that you feel comfortable voicing your opinions, giving
+Our community meetings are on Wednesday at 1PM EST/EDT. You can find the video
+conference link [here](https://meet.google.com/hys-ubrv-bsd). You can find the
+agenda [here](https://drive.google.com/drive/folders/1khHMD7JKt-kNAkbLgiIfwDbJYAhHMkyg?usp=sharing) 
+as well as the meeting notes [here](https://drive.google.com/drive/folders/1rVrheoO-nfUI7JdPEgCqM39rl59KrPBk?usp=sharing).
+If you have a topic you would like to discuss, please put it on the agenda as 
+well as an time estimate. Community meetings will be recorded
```
Comment (sriumcp) [inline mkdocs/docs/contributing/overview.md]: as well as an time estimate -> as well as a time estimate

**Review by sriumcp — CHANGES_REQUESTED** [review]
minor fix requested

**Review by sriumcp — APPROVED** [review]
LGTM

**Pull request merged**